bigtable/tests/unit/test_table.py

```diff
@@ -16,6 +16,65 @@
 import unittest
 
 
+class Test___mutate_rows_request(unittest.TestCase):
+
+    def _call_fut(self, table_name, rows):
+        from google.cloud.bigtable.table import _mutate_rows_request
+
+        return _mutate_rows_request(table_name, rows)
+
+    def test__mutate_rows_too_many_mutations(self):
+        from collections import namedtuple
+        from google.cloud.bigtable._generated.data_pb2 import Mutation
+        from google.cloud.bigtable.row import DirectRow
+        import google.cloud.bigtable.table as table_module
+        from google.cloud.bigtable.table import TooManyMutationsError
+
+        table = namedtuple('Table', ['name'])
+        table.name = 'table'
+        table_module._MAX_BULK_MUTATIONS = 3
+        mutation = Mutation()
+        rows = [DirectRow(row_key=b'row_key', table=table),
+                DirectRow(row_key=b'row_key_2', table=table)]
+        rows[0]._pb_mutations = [mutation, mutation]
+        rows[1]._pb_mutations = [mutation, mutation]
+        with self.assertRaises(TooManyMutationsError):
+            self._call_fut('table', rows)
+
+
+class Test__check_row_table_name(unittest.TestCase):
+
+    def _call_fut(self, table_name, row):
+        from google.cloud.bigtable.table import _check_row_table_name
+
+        return _check_row_table_name(table_name, row)
+
+    def test__check_rows_wrong_table_name(self):
+        from collections import namedtuple
+        from google.cloud.bigtable.table import TableMismatchError
+        from google.cloud.bigtable.row import DirectRow
+
+        table = namedtuple('Table', ['name'])
+        table.name = 'table'
+        row = DirectRow(row_key=b'row_key', table=table)
+        with self.assertRaises(TableMismatchError):
+            self._call_fut('other_table', row)
+
+
+class Test__check_row_type(unittest.TestCase):
+    def _call_fut(self, table_name, row):
+        from google.cloud.bigtable.table import _check_row_type
+
+        return _check_row_type(table_name, row)
+
+    def test__check_rows_wrong_row_type(self):
```

Comment:
You can just call this: `test_wrong_row_type` (the class name already tells us that we are `Test__check_row_type`)